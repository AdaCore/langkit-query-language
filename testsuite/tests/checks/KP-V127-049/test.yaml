driver: 'checker'
rule_name: KP_V127_049
project: 'prj.gpr'
