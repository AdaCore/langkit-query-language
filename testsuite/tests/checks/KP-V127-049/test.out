gen_cmp.adb:29:32: rule violation: possible occurrence of KP V127-049
29 |                                ">"          => Less_Than);  --  FLAG
   |                                ^^^^^^^^^^^^^^^^^^^^^^^^^

gen_cmp.adb:31:58: rule violation: possible occurrence of KP V127-049
31 |    procedure Flag2 is new Gen (Integer, Alpha, My_Array, Less_Than);  --  FLAG
   |                                                          ^^^^^^^^^

